docs/content/dagster-cloud/developing-testing/environment-variables-and-secrets.mdx

```diff
@@ -0,0 +1,335 @@
+      </td>
+      <td>
+        <ul
+          style={{
+            marginTop: "0px",
+          }}
+        >
+          <li
+            style={{
+              marginTop: "0px",
+            }}
+          >
+            TODO: not sure if we're keeping this method
+          </li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/dagster-cloud/developing-testing/environment-variables/setup-for-serverless-deployments#passing-environment-variables-using-the-dagster-cloud-cli">
+          dagster-cloud CLI
+        </a>
+      </td>
+      <td>
+        Environment variables are passed in using the <code>--env</code> flag in
+        the CLI. <strong>Supported for Serverless deployments only</strong>.
+      </td>
+      <td>
+        <ul
+          style={{
+            marginTop: "0px",
+          }}
+        >
+          <li
+            style={{
+              marginTop: "0px",
+            }}
+          >
+            You're using{" "}
+            <a href="/dagster-cloud/deployment/serverless">
+              Serverless deployment
+            </a>
+          </li>
+          <li>You're not using Dagster's GitHub app</li>
+          <li>
+            You want to manage environment variables outside of Dagster Cloud
+          </li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/dagster-cloud/developing-testing/environment-variables/setup-for-hybrid-deployments">
+          Agent configuration
+        </a>
+      </td>
+      <td>
+        Environment variables are defined in the agent's configuration in your
+        project's <code>dagster_cloud.yaml</code> file.{" "}
+        <strong>Supported for Hybrid deployments only</strong>.
+      </td>
+      <td>
+        <ul
+          style={{
+            marginTop: "0px",
+          }}
+        >
+          <li
+            style={{
+              marginTop: "0px",
+            }}
+          >
+            You're using{" "}
+            <a href="/dagster-cloud/deployment/hybrid">Hybrid deployment</a>
+          </li>
+          <li>
+            You want to manage environment variables outside of Dagster Cloud
+          </li>
+        </ul>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Storage and encryption
+
+This may only apply to management using the UI. Will need to confirm once details are finalized.
+
+<TODO></TODO>
+
+### Reserved variables
+
+[Built-in Dagster Cloud environment variables](#built-in-environment-variables) are reserved and are unavailable for use. If used, the following error will be raised: <TODO></TODO>
+
+---
+
+## Managing environment variables
+
+The simplest way to manage environment variables is to use Dagster Cloud's built-in secrets manager, which allows you to create and manage environment variables right in the UI. If you prefer not to use Dagster Cloud's secrets manager, you can use one of [the other methods](#using-alternative-methods) supported for your deployment type.
```

Comment:
we could mention that the other options that don't require you to send secrets to Dagster Cloud since the agent pulls them from the user cluster (kind of like serverless vs. hybryd)